src/App.vue

```diff
@@ -7,7 +7,7 @@
     <the-header />
 
     <!-- Page content -->
-    <div class="w-full mx-auto container pb-3 sm:p-3">
+    <div class="w-full px-6 sm:mx-auto sm:container pb-3 sm:p-3">
```

Comment:
In the Figma mockups, the px is set to 16px:
![image](https://user-images.githubusercontent.com/77116011/123809936-04284300-d8c0-11eb-9d02-d74d02b37c27.png)

To get this 16px we can use px-4 insteaad of px-6.